packages/v-protocol/src/parser.ts

```diff
@@ -291,8 +291,12 @@ export class Parser {
   private parseField(): Field {
     const name = this.reader.cstring()
     const tableID = this.reader.uint64()
-    const schemaName = this.reader.cstring()
-    const tableName = this.reader.cstring()
+    var schemaName = ""
```

Comment:
Was this resulting in the packet reading getting mangled in some case (invalid tableID?)

Reply:
yes exactly. Wasn't playing nicely with queries that were returning literals, which would get resolved later in the parsing, but caused a few metadata fields like typeID to be wrong